packages/blaze-tx/src/tx.ts

```diff
@@ -555,8 +555,7 @@ export class TxBuilder {
    * @returns {bigint} The minimum ada required for the output.
    */
   private calculateMinAda(output: TransactionOutput): bigint {
```

Comment:
We should probably get rid of this, and just require that the protocol parameters is passed; having hard coded defaults is a great place for bugs to creep in